src/lib/styles/foundation/partyColors.scss

```diff
@@ -0,0 +1,447 @@
+
+@mixin default-party-colors-uk {
+    
+    --con: #0079B7;
+    --lab: #C70000;
+    --libdem: #E05E00;
+    --green: #33A22B;
+    --ukip: #BB3B80;
+    --snp: #F5DC00;
+    --dup: #8B0000;
+    --alliance: #C9BB19;
+    --uup: #004975;
+    --sdlp: #23B4A9;
+    --tig: #333333;
+    --plaid: #135E58;
+    --sinnfein: #22874D;
+    --reform: #13bece;
+    --ind: #333333;
+    --other: #848484;
+    --speaker: #767676;
+    --change: #333;
+
+    --undeclared: #848484;
+  }
+  
+@mixin dark-mode-party-colors-uk {
+
+    --con: #009ae1;
+    --lab: #dc2e1c;
+    --libdem: #FF7F0F;
+    --green: #39A566;
+    --ukip: #DC559D;
+    --snp: #F5DC00;
+    --dup: #B23C2D;
+    --alliance: #ab9f00;
+    --uup: #346896;
+    --sdlp: #28b8ad;
+    --tig: #333333;
+    --plaid: #37716b;
+    --sinnfein: #22874D;
+    --reform: #3DBBE2;
+    --ind: #A1A1A1;
+    --other: #707070;
+    --speaker: #707070;
+    --change: #494949;
+
+    --undeclared: #848484;
+  
+  }
+  
+/* BACKGROUND-COLOR -------------------------------------------- */ 
+
+.bg-color--con {
```

Comment:
If there's a better way of doing this - instead of all this repetition - i'm very open to it

Reply:
I think we can generate it like some of the other styles. I'll pick that up at a later time though